src/main/java/com/artipie/npm/proxy/RxNpmProxyStorage.java

```diff
@@ -174,6 +180,7 @@ public Completable save(final NpmAsset asset) {
     private static String packageMetadata(final NpmPackage pkg) {
         final JsonObject json = new JsonObject();
         json.put("last-modified", pkg.lastModified());
+        json.put("last-updated", DateTimeFormatter.ISO_OFFSET_DATE_TIME.format(pkg.lastUpdated()));
```

Comment:
@brastak I think we can replace methods `lastModified` and `lastUpdated` with method `JsonObject metadata()`. This way we would hide internal structure and we would avoid unnecessary conversion `json` -> `dto` -> `json`. For reading of metadata we could create a inner static class. Or we can have `Metadata metadata()` method in `NpmPackage` that will have `lastModified`, `lastUpdated` and `asJson` methods. WDYT?